### ISSUE TITLE
Unknown mod conflict

### ISSUE THREAD

**Issue by matthewjessee16** [BarryCarlyon/FS22_ExtraGears#15]
this mod is broken.

**Comment by BarryCarlyon** [comment]
What is/isn't working for you?
Does the UI draw to the screen and it's just not working as expected.
Was it working before and now is not working?

What game version do you have? What platform are you on.

Do you have another other mods installed that might interfere with how a vehicle drives?

**Comment by matthewjessee16** [comment]
first off i cant select gears with my logitech g29 gear shifter.
it draws to the screen but it doesnt show me selected gears.
i am on pc with game version 1.8.2.0 on steam.
i am currently only using vehicle control addon and enhanced vehicles.

**Comment by BarryCarlyon** [comment]
You did bind your shifter to the relevant controls in the controls editor?
Normally you would potentially use the Extra Gear binds not the default/core game binds.

Extra Gears has it's own binds in the menu

https://github.com/BarryCarlyon/FS22_ExtraGears/blob/main/screenshots/screenshot1.png

**Comment by matthewjessee16** [comment]
yes and it still doesnt work

**Comment by BarryCarlyon** [comment]
Ok I'll do some testing and have a look (my windows machine is currently down so I can't do some decent testing till later this week)

**Comment by matthewjessee16** [comment]
could i send my log?

**Comment by BarryCarlyon** [comment]
Sure that would be useful, should be able to drag/drop to this issue and it'll upload to here.

**Comment by matthewjessee16** [comment]
[log.txt](https://github.com/BarryCarlyon/FS22_ExtraGears/files/10540027/log.txt)

**Comment by matthewjessee16** [comment]
can you look?

**Comment by BarryCarlyon** [comment]
There is no obvious error or bug in the log that I can see. So I will need to investigate further. But it'll have to wait till later this week as my windows computer is down and I'm waiting for parts.

**Comment by matthewjessee16** [comment]
Have you fixed this yet?

**Comment by BarryCarlyon** [comment]
As I mentioned I am waiting for parts to repair my windows computer so I may debug and solve the issue.

There is no update on the issue I will update you when there is an update on the issue

**Comment by matthewjessee16** [comment]
Okay, I’m just excited.

**Comment by matthewjessee16** [comment]
Have you fixed this yet?

**Comment by BarryCarlyon** [comment]
Using this contol configuration

![image](https://user-images.githubusercontent.com/20999/218258968-680a706f-4241-4e22-af3e-4beba9e4ed0f.png)

In a game that has no other mods active, and the current version from mod hub, extra gears is working for me.
Using a logitch g920. (control type should not matter)

So I can't see anything wrong as the mod is working fine.

> i am currently only using vehicle control addon and enhanced vehicles.

I will grab these mods and test with them, ~~I suspect VCA is conflicting~~.
It's not, I installed VCA from ModHub and ExtraGears is working fine.

Can you link me the "enhanced vehicles" on modhub to test with as I cannot find it.

**Comment by matthewjessee16** [comment]
Enhanced vehicles is on github

**Comment by BarryCarlyon** [comment]
Can you link to it please?

**Comment by matthewjessee16** [comment]
https://github.com/ZhooL/FS22_EnhancedVehicle

**Comment by BarryCarlyon** [comment]
I am unable to find an issue.

With the following three mods installed

- Extra Gears
- vehicle control addon
- enhanced vehicles.

It's working for me without any issues.

> first off i cant select gears with my logitech g29 gear shifter.
> it draws to the screen but it doesnt show me selected gears.

Have you bound a +6'er button to see if it'll cyle the + numbers to check the + rotation is working?

Having bound 1-6 (or 1-8) in the extra gears section did you check for an unbind the default gear controls in case they are bound to your shifter and conflicting with extra gears?

Are gears changing at all or are you stuck in neutral?

**Comment by matthewjessee16** [comment]
It’s just stuck in neutral

**Comment by BarryCarlyon** [comment]
![image](https://user-images.githubusercontent.com/20999/218265145-cd0c921a-a7a0-42b3-8c49-7fea00561c55.png)

You are driving in `manual` or `manual with clutch`?

**Comment by matthewjessee16** [comment]
Manual

**Comment by BarryCarlyon** [comment]
![image](https://user-images.githubusercontent.com/20999/218265390-1998b7c8-7921-4513-935d-3ae3768a9334.png)

 and the default gamepad controls are unbound?

**Comment by matthewjessee16** [comment]
Yes

**Comment by BarryCarlyon** [comment]
And you are driving a "Default/base game" vehicle and not a modded one? (should work with modded but wise to check)

**Comment by matthewjessee16** [comment]
Default

**Comment by BarryCarlyon** [comment]
Close the game if open
Open the game
Open your save
enter a vehicle
save the game
close the game

send me that log file

**Comment by matthewjessee16** [comment]
it works fine with only dlc and the three mods

**Comment by BarryCarlyon** [comment]
yeah I'm taking another look at your earlier log. It notes a number of other mods I'm wondering if there is another mod thats is inteferrin with the normal operation of mods in Farming Simulator.

So you have another mod that is breaking something internally with an override of the core game.

**Comment by matthewjessee16** [comment]
could it be the sim dashboard interface?

**Comment by BarryCarlyon** [comment]
> Close the game if open Open the game Open your save enter a vehicle save the game close the game
> 
> send me that log file

Do this in the version/setup that is broken so I can have a fresh log to poke around with

**Comment by matthewjessee16** [comment]
[log.txt](https://github.com/BarryCarlyon/FS22_ExtraGears/files/10713940/log.txt)

**Comment by BarryCarlyon** [comment]
you did load up your world/save and jump into/out of a vehicle?

**Comment by matthewjessee16** [comment]
i forgot

**Comment by matthewjessee16** [comment]
[log.txt](https://github.com/BarryCarlyon/FS22_ExtraGears/files/10713964/log.txt)

**Comment by BarryCarlyon** [comment]
I'm really not sure which other mod is conflicting and preventing the mod operating correctly.
if you want to step thru and figure out which one stops things working then I can debug from there.

**Comment by matthewjessee16** [comment]
i thibk it might be stuff from hof bergmann

**Comment by matthewjessee16** [comment]
i removed that stuff and it works fine

**Comment by BarryCarlyon** [comment]
You have a link to that?

**Comment by matthewjessee16** [comment]
I think we can close this as it was the cause

**Comment by BarryCarlyon** [comment]
well i could potentially fix the mod/find the issue so it's compatible with the hof bergmann stuff?

**Comment by matthewjessee16** [comment]
I’ll send it later, as I’ve got work

**Comment by matthewjessee16** [comment]
Would I happen to be able to use the change gear group bindings?

**Comment by BarryCarlyon** [comment]
you should be able to configure the controls to your liking.

I don't remember the setup for a "change gear" but you should be able to set things up to exactly what you want.
This sort of layout _sim wise_ feels more geared to having two shifters plugged in so you can direct drive which group you are in

**Comment by Nightcloak** [comment]
Hof Bergmann uses a custom type of vehicle as a sort of production (stationary woodchipper and a haycrane), causes issues with a few mods that change vehicles
